src/components/TableOfContents/index.tsx

```diff
@@ -44,15 +44,11 @@ const TableOfContents: React.FC<IProps> = ({
 
   if (!isMobile) {
     const getTitleIds = (items: Array<ToCItem>, depth: number): void => {
```

Comment:
should be moved to `src/lib/utils/toc.ts`

Reply:
See note below... I'm also realizing that this function uses locally scoped variables (ie. maxDepth) and uses side effects to push to the `titleIds` array. Would suggest we address relocating/refactoring this to a separate PR.

Reply:
Ok let's leave this here